comps/comps-foreman-plugins-rhel7.xml

```diff
@@ -174,44 +174,9 @@
       <packagereq type="default">rubygem-algebrick-doc</packagereq>
       <packagereq type="default">rubygem-faraday-doc</packagereq>
       <packagereq type="default">rubygem-fast_gettext-doc</packagereq>
-      <packagereq type="default">rubygem-mysql2-doc</packagereq>
```

Comment:
It would be good to already get these comps cleanups in, regardless of this package.